src/RoverExplorer/RoverExplorer.tsx

```diff
@@ -1,11 +1,27 @@
 import React from "react";
 import styles from './RoverExplorer.module.scss';
+import {
+    BrowserRouter as Router,
+    Switch,
+    Route,
+    Link
+} from "react-router-dom";
+import {Button} from "semantic-ui-react";
 
 
-export function RoverExplorer() {
+export function RoverExplorer () {
     return (
-        <div className={styles.roverExplorer}>
-            <h2>Rover Explorer</h2>;
-        </div>
-    );
-}
\ No newline at end of file
+    <section>
+    <div className ={styles.Body}>
+    <header className= {styles.Header}><div className ={styles.HeaderText}>Rover Explorer</div></header>
+    <div className={styles.GridContainer}>
+      <Button as={Link} to="/timeline/perserverance" className={styles.GridItem1}>perserverance</Button>
```

Comment:
why are we using a library for this link?

Reply:
Couldn’t find another way for the divs to work like buttons